axlearn/common/input_tf_data.py

```diff
@@ -863,6 +868,79 @@ def fn(ds: tf.data.Dataset) -> tf.data.Dataset:
     return fn
 
 
+def per_feed_batch(
+    feed_batch_size: int,
+    *,
+    is_training: bool,
+    pad_example_fn: PadExampleFn,
+    prefetch_buffer_size: Optional[int] = None,
+    post_batch_processor: Optional[ConfigOr[DatasetToDatasetFn]] = None,
+    repeat: Optional[int] = None,
+) -> DatasetToDatasetFn:
+    """Returns a function that generates a tf.data.Dataset object.
```

Comment:
nit -- the comment seems to suggest it returns a source instead of a processor.

Reply:
Done.